src/plugins/wb-disable/wb-disable-fr.hbs

```diff
@@ -0,0 +1,24 @@
+---
+{
+"title": "HTML simplifié",
+"language": "fr",
+"category": "Plugins",
```

Comment:
Voici le correctif pour l'erreur relevé par le link checker de notre CI script

```suggestion
"category": "Plugiciels",
```

Reply:
Unfortunate that `grunt dist` or `grunt test` don't notice that error. It is fixed now!